### PR TITLE
Add dashboard for blackbox exporter as configmap

### PR DESCRIPTION
**Pull request by michal-gubricky**
Related to https://github.com/SovereignCloudStack/issues/issues/480

### CODE REVIEW

File: mvp-0/blackbox-dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,942 @@
...
+            "type": "dashboard"
+          }
+        ]
+      },
+      "description": "Blackbox exporter HTTP prober dashboard",
+      "editable": true,
+      "fiscalYearStartMonth": 0,
+      "gnetId": 13659,
+      "graphTooltip": 0,
+      "id": 124,
```
Comment (chess-knight) [inline mvp-0/blackbox-dashboard.yaml:30]:
```suggestion
      "id": null,
```

File: mvp-0/blackbox-dashboard.yaml [hunk 2]
```diff
@@ -0,0 +1,942 @@
...
+            "tagValuesQuery": "",
+            "tagsQuery": "",
+            "type": "query",
+            "useTags": false
+          }
+        ]
+      },
+      "time": {
+        "from": "2023-11-22T14:44:37.777Z",
+        "to": "2023-11-22T14:49:37.777Z"
```
Comment (chess-knight) [inline mvp-0/blackbox-dashboard.yaml:934]:
```suggestion
        "from": "now-5m",
        "to": "now"
```

**Review by chess-knight — APPROVED** [review]
Overall LGTM, just minor things...

File: mvp-0/blackbox-dashboard.yaml (hunk 2 above)
Comment (michal-gubricky) [reply mvp-0/blackbox-dashboard.yaml]: Fixed

File: mvp-0/blackbox-dashboard.yaml (hunk 1 above)
Comment (michal-gubricky) [reply mvp-0/blackbox-dashboard.yaml]: Fixed

**Pull request merged**